Makefile

```
@@ -1,13 +1,14 @@
 # Image URL to use all building/pushing image targets
 TAG ?= $(shell git describe --tags --abbrev=0 --match '[0-9].*[0-9].*[0-9]' 2>/dev/null )
 IMG ?= banzaicloud/istio-operator-v2:$(TAG)
-# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
 
 GOLANGCI_VERSION = 1.39.0
 LICENSEI_VERSION = 0.3.1
 KUBEBUILDER_VERSION = 2.3.2
 KUSTOMIZE_VERSION = 4.1.2
+ISTIO_VERSION = 1.9.3
```

I'd suggest going for 1.10 right away, so use the latest 1.10 tag here.
```suggestion
ISTIO_VERSION = 1.10.0-alpha.1
```